test/cpp/jit/test_schema_info.cpp

```diff
@@ -173,6 +173,48 @@ TEST(SchemaInfoMayAliasTest, AliasingInputOutput) {
       {c10::SchemaArgType::input, 1}, {c10::SchemaArgType::output, 0}));
 }
 
+TEST(SchemaInfoMayAliasTest, MultipleWildcardInputs) {
+  SchemaInfo schema(
+      "aten::test.Tensor(Tensor(a) a, Tensor(*) b, Tensor(*) c) -> (Tensor(a), Tensor(*))");
```

Comment:
mostly just curious, would this handle `fn(Tensor(a -> *) x) -> (Tensor(a), Tensor(*))` correctly?

Reply:
I don't think so. I'm not really considering cases where (a -> *) exists but that might be something that I can implement in a later pr.